lib/topological_inventory/orchestrator/deployment_config.rb

```diff
@@ -72,6 +72,28 @@ def delete_in_openshift
         logger.info("[OK] Deleted DeploymentConfig #{self}")
       end
 
+      def update_image(image)
+        logger.info("Updating DeploymentConfig Image #{self}")
+
+        # This is ugly, but its a JSON patch we're sending.
+        patch = {
+          :spec => {
+            :template => {
+              :spec => {
+                :containers => [
+                  :name  => openshift_object.spec.template.spec.containers.first.image,
```

Comment:
Could we just use the `image` method here too?

Reply:
I messed up there, it was late. I meant to pull the name of the container. Also updated the name of the var from `image` to `new_image`.